apps/guardian-ui/src/utils/network.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+export const getNetworkIndicator = (network: string, bitcoinRpcUrl: string) => {
```

Comment:
wondering if we have need for a similar util/ux in gateway UI?

Reply:
no that one already shows the network, can add in a separate PR if we want it there too

Reply:
added it to the guardian dashboard at least